# HG changeset patch
# User Mike Miller <mtmiller@ieee.org>
# Date 1330473723 18000
#      Tue Feb 28 19:02:03 2012 -0500
# Node ID 2258a0b73eb80bb34fd4c157537be9dd7b4d009c
# Parent  933101fd5cbbf51da303d79bcd8a55f1d262d39f
Fix path to missing helper script

diff --git a/m4/acinclude.m4 b/m4/acinclude.m4
--- a/m4/acinclude.m4
+++ b/m4/acinclude.m4
@@ -473,33 +473,33 @@ AC_DEFUN([OCTAVE_PROG_FLEX], [
   AC_PROG_LEX
   case "$LEX" in
     flex*)
       LFLAGS="-I"
       AC_MSG_RESULT([defining LFLAGS to be $LFLAGS])
       LEXLIB=
     ;;
     *)
-      LEX='$(top_srcdir)/missing flex'
+      LEX='$(top_srcdir)/build-aux/missing flex'
       warn_flex="I didn't find flex, but it's only a problem if you need to reconstruct lex.cc"
       AC_MSG_WARN([$warn_flex])
     ;;
   esac
   AC_SUBST(LFLAGS)
 ])
 dnl
 dnl Check for bison
 dnl
 AC_DEFUN([OCTAVE_PROG_BISON], [
   AC_PROG_YACC
   case "$YACC" in
     bison*)
     ;;
     *)
-      YACC='$(top_srcdir)/missing bison'
+      YACC='$(top_srcdir)/build-aux/missing bison'
       warn_bison="I didn't find bison, but it's only a problem if you need to reconstruct parse.cc"
       AC_MSG_WARN([$warn_bison])
     ;;
   esac
 ])
 dnl
 dnl What pager should we use?
 dnl
@@ -548,17 +548,17 @@ AC_SUBST(GNUPLOT)
 ])
 dnl
 dnl Is gperf installed?
 dnl
 dnl OCTAVE_PROG_GPERF
 AC_DEFUN([OCTAVE_PROG_GPERF], [
   AC_CHECK_PROG(GPERF, gperf, gperf, [])
   if test -z "$GPERF"; then
-    GPERF='$(top_srcdir)/missing gperf'
+    GPERF='$(top_srcdir)/build-aux/missing gperf'
     warn_gperf="I didn't find gperf, but it's only a problem if you need to reconstruct oct-gperf.h"
     AC_MSG_WARN([$warn_gperf])
   fi
   AC_SUBST(GPERF)
 ])
 dnl
 dnl Is ghostscript installed?
 dnl
@@ -569,30 +569,30 @@ AC_DEFUN([OCTAVE_PROG_GHOSTSCRIPT], [
       gs_names="gswin32c gs mgs"
     ;;
     *)
       gs_names="gs"
     ;;
   esac
   AC_CHECK_PROGS(GHOSTSCRIPT, [$gs_names])
   if test -z "$GHOSTSCRIPT"; then
-    GHOSTSCRIPT='$(top_srcdir)/missing gs'
+    GHOSTSCRIPT='$(top_srcdir)/build-aux/missing gs'
     warn_ghostscript="I didn't find ghostscript, so reconstructing figures for the manual will fail, and saving graphics in some output formats will fail when using Octave"
     AC_MSG_WARN([$warn_ghostscript])
   fi
   AC_SUBST(GHOSTSCRIPT)
 ])
 dnl
 dnl Is texi2dvi installed?
 dnl
 dnl OCTAVE_PROG_TEXI2DVI
 AC_DEFUN([OCTAVE_PROG_TEXI2DVI], [
   AC_CHECK_PROG(TEXI2DVI, texi2dvi, texi2dvi, [])
   if test -z "$TEXI2DVI"; then
-    TEXI2DVI='$(top_srcdir)/missing texi2dvi'
+    TEXI2DVI='$(top_srcdir)/build-aux/missing texi2dvi'
     warn_texi2dvi="I didn't find texi2dvi, but it's only a problem if you need to reconstruct the DVI version of the manual"
     AC_MSG_WARN([$warn_texi2dvi])
   fi
   AC_SUBST(TEXI2DVI)
 ])
 dnl
 dnl Is texi2pdf installed?
 dnl
@@ -605,17 +605,17 @@ AC_DEFUN([OCTAVE_PROG_TEXI2PDF], [
     if test -n "$TEXI2DVI"; then
       TEXI2PDF="$TEXI2DVI --pdf"
       missing=false;
     fi
   else
     missing=false;
   fi
   if $missing; then
-    TEXI2PDF='$(top_srcdir)/missing texi2pdf'
+    TEXI2PDF='$(top_srcdir)/build-aux/missing texi2pdf'
     warn_texi2pdf="I didn't find texi2pdf, but it's only a problem if you need to reconstruct the PDF version of the manual"
     AC_MSG_WARN([$warn_texi2pdf])
   fi
   AC_SUBST(TEXI2PDF)
 ])
 dnl
 dnl See if the C++ library is ISO compliant.
 dnl FIXME: This is obviously very simplistic, and trivially fooled.
